test/parallel/test-debugger-list-no-module-wrapper.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const common = require('../common');
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+const fixture = path.join(
+  common.fixturesDir,
+  'debugger-repeat-last.js'
+);
+
+const args = [
+  'debug',
+  `--port=${common.PORT}`,
+  fixture
+];
+
+const proc = spawn(process.execPath, args, { stdio: 'pipe' });
+proc.stdout.setEncoding('utf8');
+
+var stdout = '';
+
+var sentCommand = false;
+var sentEmpty = false;
+var sentExit = false;
+
+proc.stdout.on('data', (data) => {
+  stdout += data;
+  if (!sentCommand && stdout.includes('> 1')) {
+    return sentCommand = true;
+  }
+
+  if (!sentEmpty) {
+    setImmediate(() => {proc.stdin.write('list(10)\n');});
+    return sentEmpty = true;
+  }
+  if (!sentExit && sentCommand && sentEmpty) {
+    setTimeout(() => {proc.stdin.write('\n\n\n.exit\n\n\n');}, 1);
```

Comment:
i copied and pasted from another test, so to be honest,  thats what happened.  is there a preferences for me to switch too